src/AddTodoForm.jsx

```diff
@@ -14,9 +14,9 @@ function AddTodoForm({ onAddTodo }) {
   const handleAddTodo = (event) => {
     event.preventDefault();
     console.log(todoTitle)
-    const Todo = {title:todoTitle, id: Date.now()}
-    console.log(Todo.id)
-    onAddTodo(Todo);
+    const todo = {title:todoTitle, id: Date.now()}
```

Comment:
great! you almost complete the stretch goal. 
Just one issue, we cannot see todo titles from Airtable. 
<img width="675" alt="image" src="https://github.com/user-attachments/assets/b12ecfe8-4ddc-4a93-abd1-dac9b88c3e93">

Be sure that you've read all explanation for the stretch goal regarding Date.now() function
<img width="609" alt="image" src="https://github.com/user-attachments/assets/47daaa73-378f-456e-94f3-86130da53091">

small hint: 
here the data you received from the fetch function 
<img width="422" alt="image" src="https://github.com/user-attachments/assets/0fb3cb4e-6064-45e7-ae13-43aca147dcd6">